src/elexmodel/models/GaussianElectionModel.py

```diff
@@ -73,18 +73,18 @@ def get_unit_prediction_intervals(self, reporting_units, nonreporting_units, alp
         lower = prediction_intervals.lower - lower_correction
         upper = prediction_intervals.upper + upper_correction
 
-        # un-normalize residuals
-        lower *= nonreporting_units[f"last_election_results_{estimand}"]
-        upper *= nonreporting_units[f"last_election_results_{estimand}"]
+        # if estimand == "turnout":
+        #     lower = (lower + nonreporting_units["last_election_share_turnout"]) * nonreporting_units["total_age_voters"]
+        #     upper = (upper + nonreporting_units["last_election_share_turnout"]) * nonreporting_units["total_age_voters"]
+
+        # else:
+        #     lower = (lower + nonreporting_units[f"last_election_share_{estimand}"]) * self.raw_turnout_preds
+        #     upper = (upper + nonreporting_units[f"last_election_share_{estimand}"]) * self.raw_turnout_preds
 
         # move from residual to vote space
         # max with nonreporting results so that bounds are at least as large as the # of votes seen
-        lower = np.maximum(
-            lower + nonreporting_units[f"last_election_results_{estimand}"], nonreporting_units[f"results_{estimand}"]
-        )
-        upper = np.maximum(
-            upper + nonreporting_units[f"last_election_results_{estimand}"], nonreporting_units[f"results_{estimand}"]
-        )
+        lower = np.maximum(lower, nonreporting_units[f"results_{estimand}"])
```

Comment:
do you mind explaining what is happening here? Especially why this is different from the parametric model?